.github/workflows/update_readmes.yml

```diff
@@ -0,0 +1,46 @@
+name: Update Haystack version in social cards
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update-gh-org-readme:
+    name: Update GitHub Organization Readme
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger update in GitHub Org Readme
+        env:
+          VERSION: ${{ github.event.release.tag_name }}
```

Comment:
I would temporarily add manual invocation via dispatch and hardcode something here in order to test this PR. Then we can revert and merge.